h2o-docs/src/product/data-science/stacked-ensembles.rst

```diff
@@ -65,55 +65,79 @@ Before training a stacked ensemble, you will need to train and cross-validate a
 
--  `metalearner_transform <algo-params/metalearner_transform.html>`__: (Optional) Specify the transformation used on predictions from the base models in order to make a level one frame. Options include:
+-  `metalearner_transform <algo-params/metalearner_transform.html>`__: Specify the transformation used on predictions from the base models in order to make a level one frame. Options include:
 
  - ``"NONE"`` (no transform applied)
  - ``"Logit"`` (applicable only to classification tasks, use logit transformation on the predicted probabilities)
 
--  `offset_column <algo-params/offset_column.html>`__: (Optional; Availability depends on the ``metalearner_algorithm``) Specify a column to use as the offset.
+-  `offset_column <algo-params/offset_column.html>`__: (Availability depends on the ``metalearner_algorithm``) Specify a column to use as the offset.
 
--  `weights_column <algo-params/weights_column.html>`__: (Optional) Specifies a column with observation weights. Giving some observation a weight of zero is equivalent to excluding it from the dataset; giving an observation a relative weight of 2 is equivalent to repeating that row twice. Negative weights are not allowed.
+-  `weights_column <algo-params/weights_column.html>`__: Specifies a column with observation weights. Giving some observation a weight of ``0`` is equivalent to excluding it from the dataset; giving an observation a relative weight of ``2`` is equivalent to repeating that row twice. Negative weights are not allowed.
 
--  **keep_levelone_frame**: (Optional) Keep the level one data frame that's constructed for the metalearning step. Defaults to False.
+-  `seed <algo-params/seed.html>`__: Seed for random numbers; passed through to the metalearner algorithm. Defaults to ``-1`` (time-based random number).
 
--  `max_runtime_secs <algo-params/max_runtime_secs.html>`__: (Optional) Maximum allowed runtime in seconds for the metalearner model training. Use 0 to disable the time limit. Defaults to 0.
+- **score_training_samples**: Specify the number of training set samples for scoring. The value must be :math:`\geq` 0. To use all training samples, enter ``0``. This value defaults to ``10000``.
 
--  `seed <algo-params/seed.html>`__: (Optional) Seed for random numbers; passed through to the metalearner algorithm. Defaults to -1 (time-based random number).
+-  **keep_levelone_frame**: Keep the level one data frame that's constructed for the metalearning step. Defaults to ``False``.
 
 -  `export_checkpoints_dir <algo-params/export_checkpoints_dir.html>`__: Specify a directory to which generated models will automatically be exported.
 
+- `auc_type <algo-params/auc_type.html>`__: Set the default multinomial AUC type. Must be one of:
+
+    - ``"AUTO"`` (default)
+    - ``"NONE"``
+    - ``"MACRO_OVR"``
+    - ``"WEIGHTED_OVR"``
+    - ``"MACRO_OVO"``
+    - ``"WEIGHTED_OVO"``
+
+Hyperparameters
+'''''''''''''''
```

Comment:
I'm not sure if the "Hyperparameters" is the proper term to use here - I'd expect `metalearner_algorithm`, `metalearner_transform` etc. to be included in hyperparameters. Also consistency -  I noticed that GLM has `seed` included in hyperparameters while SE has `seed` in the "Common Parameters" section.

Maybe just having "Common Parameters" would be better than splitting it into two catergories, especially if we'd have just one parameter as a "hyperparameter".

Reply:
Ah, ``seed`` was my mistake -- thank you. However, the [metalearner_algorithm](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/metalearner_algorithm.html) and[ metalearner_transform](https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/metalearner_transform.html) appendix pages mark them as not hyperparams. Is this information incorrect?

Reply:
If it they were just integrated in, this is what it would look like:

<img width="747" alt="Screen Shot 2022-07-20 at 8 58 10 AM" src="https://user-images.githubusercontent.com/52463461/180000972-aedf4473-8563-4015-b75b-29b035714bc3.png">

Reply:
It looks like it should be correct now.  I thought the **hyperparameter** is some arbitrary section but it is based on what can be used in the grid search. So it would be probably better to have it consistent with other algos in separate section. I'm sorry for not checking documentation more before I pointed it out as a mistake.

Reply:
Actually, it might be a good idea to outline under the Hyperparameters header _"These parameters can be used in grid search."_ because I also found it confusing at first.

Reply:
@hannah-tillman will we add the `"These parameters can be used in grid search."` here? Otherwise, the split between common and hyperparameters (those that are "gridable") looks correct to me.

Reply:
Thank you for the reminder -- adding that in right now :)